### PR TITLE
updated few topics

### PR DESCRIPTION
**Pull request by tishko0**
Recreated PR and updated topics as per your request here: https://github.com/IgniteUI/igniteui-xplat-docs/pull/528

### CODE REVIEW

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: doc/en/components/grids/_shared/live-data.md [hunk 1]
```diff
@@ -20,31 +19,39 @@ Feed the same data into the [Line Chart](../charts/types/line-chart.md) to exper
            iframe-src="{environment:dvDemosBaseUrl}/{ComponentSample}-grid-finjs"
            alt="Angular Live-data Update Example">
 </code-view>
-<!-- end: Angular -->
```
[the remark this answers is not in the document]
Comment (tishko0) [reply doc/en/components/grids/_shared/live-data.md:23]: re-added

File: doc/en/components/grids/_shared/live-data.md [hunk 2]
```diff
@@ -9,7 +9,6 @@ sharedComponents: ["Grid", "TreeGrid"]
 
 The {ComponentTitle} component is able to handle thousands of updates per second, while staying responsive for user interactions.
 
-<!-- Angular -->
```
[the remark this answers is not in the document]
Comment (tishko0) [reply doc/en/components/grids/_shared/live-data.md]: re-added

File: doc/en/components/grids/_shared/live-data.md [hunk 3]
```diff
@@ -20,31 +19,39 @@ Feed the same data into the [Line Chart](../charts/types/line-chart.md) to exper
...
 
 A service provides data to the component when the page loads, and when the slider controller is used to fetch a certain number of records. While in a real scenario updated data would be consumed from the service, here data is updated in code. This is done to keep the demo simple and focus on its main goal - demonstrate the grid performance.
 
+```Razor
+<IgbDataGrid data="data"><IgbDataGrid>
+```
+
 ```html
-<igx-grid #grid [data]="data"></igx-grid>
+<{ComponentInstance} [data]="data"></{ComponentInstance}>
```
[the remark this answers is not in the document]
Comment (tishko0) [reply doc/en/components/grids/_shared/live-data.md]: replaced

File: doc/en/components/grids/_shared/live-data.md [hunk 4]
```diff
@@ -20,31 +19,39 @@ Feed the same data into the [Line Chart](../charts/types/line-chart.md) to exper
...
+</IgbDataGrid>
+```
 
 ```html
-<igx-grid #grid [data]="data">
+<{ComponentInstance} #grid [data]="data">
     <igx-column field="price.usd"></igx-column>
-</igx-grid>
+</{ComponentInstance}>
 ```
```
[the remark this answers is not in the document]
Comment (tishko0) [reply doc/en/components/grids/_shared/live-data.md:56]: replaced

File: doc/en/components/grids/_shared/live-data.md [hunk 5]
```diff
@@ -61,24 +68,11 @@ private updateData(data: IRecord[]) {
...
-The purpose of this demo is to showcase a financial screen board with Real-time data stream using a [SignalR](https://dotnet.microsoft.com/apps/aspnet/signalr) hub back-end.
-As you can see the igxGrid component handles with ease the high-frequency updates from the server. The code for the ASP.NET Core application using SignalR could be found in this [public GitHub repository](https://github.com/IgniteUI/finjs-web-api).
-
-<code-view style="height:700px"
-           data-demos-base-url="{environment:dvDemosBaseUrl}"
-           iframe-src="{environment:dvDemosBaseUrl}/{ComponentSample}-grid-finjs-dock-manager"
-           alt="{Platform} {ComponentTitle} Live-data Update Example with a service">
-</code-view>
-<!-- end: Angular -->
-
```
[the remark this answers is not in the document]
Comment (tishko0) [reply doc/en/components/grids/_shared/live-data.md]: done

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**